README.md

```diff
@@ -895,6 +895,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Swapna Devi](https://github.com/Swapna804)
   - [Syed Sibteali Baqar](https://github.com/sibteali786)
   - [syedareehaquasar](https://github.com/syedareehaquasar)
+  - [Shariq](https://github.com/cosmicqbit)
```

Comment:
Please add your name in line 824, after Sharan C M. Let us know if you need any help. :)

Reply:
Apologies, I didn't realized that it follows alphabetical order, Anyways it's done. Thanks for your response.